src/Datasource/EntityTrait.php

```diff
@@ -1231,7 +1231,7 @@ public function source($alias = null)
      */
     public function __toString()
     {
-        return json_encode($this, JSON_PRETTY_PRINT);
+        return json_encode($this, JSON_PRETTY_PRINT | JSON_UNESCAPED_UNICODE);
```

Comment:
Do you think about something like this in `BaseLog::_format()`?
```
        $object = is_object($data);

        if ($object && $data instanceof EntityTrait) {
            return json_encode($data, JSON_UNESCAPED_UNICODE);
        }

        if ($object && method_exists($data, '__toString')) {
            return (string)$data;
        }

        if ($object && $data instanceof JsonSerializable) {
            return json_encode($data, JSON_UNESCAPED_UNICODE);
        }
```
That would leave the logging scenarios sprintf() and concatting entities into log messages unchanged (escaping unicode). Is that your intention here?